portfolio/src/main/webapp/script.js

```diff
@@ -32,8 +32,38 @@ function addRandomQuote() {
 }
 
 //use fetch to request content from server and add it to page.
-async function getMyName() {
-  const response = await fetch('/data');
-  const quote = await response.text();
-  document.getElementById('name-container').innerText = quote;
+// async function getMyName() {
+//   const response = await fetch('/data');
+//   const quote = await response.text();
+//   document.getElementById('name-container').innerText = quote;
+// }
+
+function getMessages() {
+  fetch('/data').then(response => response.json()).then((messages) => {
+    console.log(messages.message_1);
+    console.log(messages.message_2);
+    console.log(messages.message_3);
+
+    
+
+    const commentList = document.getElementById('message-container');
+    commentList.innerHTML = '';
+    commentList.appendChild(
+        createListElement('Comment 1: ' + messages.message_1));
```

Comment:
Using three variables (`message_1`, `message_2`, and `message_3`) is fine for now, because the goal of this step is to get the connection piece working. But something to think about for the next step: how do you want to handle data when you won't know ahead of time how many messages there are?